src/app/suggestion-notifications/reciter-suggestions/suggestions-notification/suggestions-notification.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { SuggestionTarget } from '../../../core/suggestion-notifications/reciter-suggestions/models/suggestion-target.model';
+import { TranslateService } from '@ngx-translate/core';
+import { SuggestionTargetsStateService } from '../suggestion-targets/suggestion-targets.state.service';
+import { NotificationsService } from '../../../shared/notifications/notifications.service';
+import { SuggestionsService } from '../suggestions.service';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'ds-suggestions-notification',
+  templateUrl: './suggestions-notification.component.html',
+  styleUrls: ['./suggestions-notification.component.scss']
+})
+export class SuggestionsNotificationComponent implements OnInit {
+
+  labelPrefix = 'mydspace.';
```

Comment:
Is this component really specific to mydspace?  Can it ever appear elsewhere?  If it can appear elsewhere, this label prefix is probably wrong.